Updated documentation for detailed processing log in KsqlDB.io project (#9349)
confluentinc/ksql: docs/reference/processing-log.md

!!! important
    In {{ site.ccloud }}, `ksql.logging.processing.rows.include` is set
    to `true`, so the default behavior is to include row data in the
    processing log. It can be configured manually when provisioning the cluster in 

>     Confluent Cloud by toggling the hide row data when provisioning 

Reviewer: ```suggestion
    {{ site.ccloud }} by toggling the hide row data when provisioning 
```

>     Confluent Cloud by toggling the hide row data when provisioning 

Reviewer: ```suggestion
    Confluent Cloud by toggling **hide row data** when provisioning 
```

>     in the UI, or setting the `log-exclude-rows` flag in the CLI

Reviewer: ```suggestion
    in the UI, or by setting the `log-exclude-rows` flag in the CLI.
```
